metaswitch/common/network_namespace.py

```diff
@@ -1,4 +1,4 @@
-# @file alarms.py
+# @file network_namespace.py
 #
```

Comment:
Signaling is American, Signalling is English. We use the American version for consistency. I'll fix it up.